app/src/main/java/org/systers/mentorship/view/fragments/MembersFragment.kt

```diff
@@ -126,6 +128,15 @@ class MembersFragment : BaseFragment() {
                                 adapter = rvAdapter
                                 isRecyclerView = true
                             }
+                        } else {
+                            if (!filterMap["location"].isNullOrEmpty()) {
+
+                                if (!membersViewModel.userList.any { (it.location)
+                                    ?.contains(filterMap["location"]!!, ignoreCase = true) == true }) {
```

Comment:
could this condition result be pulled to a variable? I don't quite understand at first what this is doing.